napari/layers/vectors/vectors.py

```diff
@@ -159,6 +172,7 @@ def __init__(
         properties=None,
         property_choices=None,
         edge_width=1,
+        vector_style='line',
```

Comment:
I know it's backwards incompatible, but I think the default should be 'triangle'.

Reply:
I don't know your (the code developers) policy about insuring backward compatibility well enough to take a stand on this matter... :sweat_smile:
@brisvag do you have any preference ?

Reply:
I agree on the triangle default, and I don't think we shoudl worry about backward compatibility here. It only changes what it looks like, but any code that worked before will still work (unless it uses private stuff), and I can't think of cases where it will change so much as to become confusing/invisible, since the length is the same and it's now just tapering off.